sormas-ui/src/main/java/de/symeda/sormas/ui/customexport/ExportConfigurationsGrid.java

```diff
@@ -30,22 +30,29 @@
 import com.vaadin.ui.themes.ValoTheme;
 
 import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.i18n.Captions;
 import de.symeda.sormas.api.i18n.I18nProperties;
 import de.symeda.sormas.api.i18n.Strings;
 import de.symeda.sormas.api.importexport.ExportConfigurationCriteria;
 import de.symeda.sormas.api.importexport.ExportConfigurationDto;
 import de.symeda.sormas.api.importexport.ExportGroupType;
 import de.symeda.sormas.api.importexport.ExportType;
+import de.symeda.sormas.api.user.UserRight;
 import de.symeda.sormas.api.utils.DataHelper;
 import de.symeda.sormas.ui.ControllerProvider;
+import de.symeda.sormas.ui.UserProvider;
 import de.symeda.sormas.ui.utils.ButtonHelper;
 
 @SuppressWarnings("serial")
 public class ExportConfigurationsGrid extends Grid<ExportConfigurationDto> {
 
 	public static final String COLUMN_ACTIONS = "actions";
+	public static final String COLUMN_ACTIONS_PUBLIC = "actionsPublic";
 
 	private Consumer<ExportConfigurationDto> exportCallback;
+	private Consumer<ExportConfigurationDto> exportCallbackSharedCustomConfiguration;
```

Comment:
`exportCallbackSharedCustomConfiguration` is does the same thing as `exportCallback` so it is not needed